### PR TITLE
added more instructions to create pipeline

### LINKED ISSUE #231: Explain kfp runtime & runtime images in `create-ai-pipeline.md` [meteor]
Data Scientists interested in cloud native development may not be familiar with runtime images and kubeflow pipeline runtimes before starting this tutorial.

It might be worth adding some descriptions about what each does, to make these concepts clear to the data science user.

Acceptance Criteria:
- [ ] Add a one line explanation for the purpose of a runtime image and a kubeflow pipeline runtime.

### LINKED ISSUE #232: add clear instructions on how to edit notebook properties in `create-ai-pipeline.md` [meteor]
In `create-ai-pipeline.md`, the [Create Elyra AI Pipeline Using the U](https://github.com/AICoE/elyra-aidevsecops-tutorial/blob/master/docs/source/create-ai-pipeline.md#create-elyra-ai-pipeline-using-the-ui)I Step 3 does not tell you that you need to find the “properties” of the step to add this info. There could be some explanation on how each field in properties needs to be configured.

Acceptance Criteria:
- [ ] Add instructions to find "properties" to add info for each notebook 
- [ ] Provide details on how to fill out each field in the properties.
- [ ] “Add comments and describe your steps (Optional).” - add clear instructions on how to add comments for nodes

### PR DESCRIPTION
**Pull request by oindrillac**
## Related Issues and Dependencies

Closes #231
Closes #232 

## This introduces a breaking change

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/source/create-ai-pipeline.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 ## Add runtime images (using UI)
 
-Now that your images are available on the registry, we need to add them to [Elyra][1] metadata:
+Now that your images are available on the registry, we need to add them to [Elyra][1] metadata. For that, we need to create a runtime image which identifies the container image that Elyra can utilize to run Jupyter notebooks. This container image contains the dependencies required for running the notebooks.
```
[the remark this answers is not in the document]
Comment (oindrillac) [reply docs/source/create-ai-pipeline.md:5]: yes, we are adding the uri of the notebook images that we created in the previous steps by creating a "runtime image" as they call it, thats how it knows where to find the container image https://elyra.readthedocs.io/en/latest/user_guide/runtime-image-conf.html#runtime-image-configuration

the next step is creating the kubeflow pipeline runtime configuration which is different from this.

File: docs/source/create-ai-pipeline.md [hunk 2]
```diff
@@ -54,6 +55,8 @@ To learn more about adding runtime images, check the following [link](https://el
 
 ### Create runtime to be used in Kubeflow pipeline (using UI)
 
+We also need to create a Kubeflow Pipeline Runtime configuration which provides Elyra access to the external Kubeflow Pipeline where we wish to run the pipeline that we are creating.
```
[the remark this answers is not in the document]
Comment (oindrillac) [reply docs/source/create-ai-pipeline.md]: done :)

File: docs/source/create-ai-pipeline.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MichaelClifford) [reply docs/source/create-ai-pipeline.md:5]: according to this link I think they call it a "runtime image configuration", and not a "runtime image". I think this edit might make the most sense. WDYT? 

```suggestion
Now that your images are available on the registry, we need to add them to [Elyra][1] metadata. For that, we need to create a runtime image configuration which identifies the container image that Elyra can pull from our registry and utilize to run the Jupyter notebooks.
```

File: docs/source/create-ai-pipeline.md [hunk 3]
```diff
@@ -118,24 +121,42 @@ To create a new pipeline from scratch, go through the following steps:
...
+
 
 <div style="text-align:center">
 <img alt="Pipeline Steps Inputs" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/AIPipelineStepInputs.png">
 </div>
 
 The notebooks can exchange data using a cloud object storage and also using the Output Files which can be used to make certain files generated during execution available to all subsequent pipeline steps.
 
-The `download_dataset` and `training` step of the pipeline have some mandatory and optional ennvironment variables for running in automation which need to be set during this step of setting up the pipeline.
+The `download_dataset` and `training` step of the pipeline have some mandatory and optional environment variables for running in automation which need to be set during this step of setting up the pipeline.
```
Comment (isabelizimm) [inline docs/source/create-ai-pipeline.md:146]: Are we able to have a screenshot of the panel with all env variables added or at least an explicit list of what environment variables you must add for the pipeline to run? Reading through this doc, there's lots of good suggestions of what to add, but its not clear what the minimal requirements are.

File: docs/source/create-ai-pipeline.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (oindrillac) [reply docs/source/create-ai-pipeline.md]: made the changes :)

File: docs/source/create-ai-pipeline.md (hunk 3 above)
Comment (oindrillac) [reply docs/source/create-ai-pipeline.md:145]: @isabelizimm thanks for pointing that out! in the section just below this we have a minimal list of variables that need to be configured. It was called "NOTE on notebook properties" but I now made it a little more clear. And added a couple images which show those environment variables.

**Review by isabelizimm — APPROVED** [review]
/lgtm 

this looks great!

**Review by chauhankaranraj — APPROVED** [review]
/lgtm :rocket:

**Pull request merged**